lib/encryptor.js

```diff
@@ -0,0 +1,19 @@
+var cipher = require('./cipher.js');
+
+exports.encryptMessageContent = encryptMessageContent;
```

Comment:
We planned to move "cipher" to separate repository later.
But encryptor.js will remain in Sailor and it will use cipher from /node_modules
Encryptor.js is only for sailor, but cipher will be included.